src/js/components/FormField/doc.js

```diff
@@ -101,7 +104,29 @@ export const themeDoc = {
   'formField.content.pad': {
     description: 'The pad of the FormField content.',
     type: 'object',
-    defaultValue: "{ horizontal: 'small', bottom: 'small' }",
+    defaultValue: 'small',
+  },
+  'formField.disabled.background.color': {
+    description: 'The color of the FormField background when it is disabled.',
+    type: "string | {'dark': string, 'light': string}",
+    defaultValue: undefined,
+  },
+  'formField.disabled.background.opacity': {
+    description: 'The opacity of the FormField background when it is disabled.',
+    type: 'string',
```

Comment:
According to https://github.com/grommet/grommet/blob/c4f9d42dc862bddde612172237a57aa7003c648a/src/js/utils/background.js#L81

I think that `background.opacity` could also be boolean and a number.